src/components/variables.scss

```diff
@@ -12,3 +12,7 @@ $plum: #c69cd0;
 
 $container-width: 880px;
 $screen-phone: 480px;
+
+// new variables
+$exciting: #311b58;
+$plain: #ffffff;
```

Comment:
- #311b58 is the updated value for $aubergine.
- please don't introduce color-variable named after purpose. the color/background of a `exciting` component is dependent on the context/contrast of a component. for example a exciting button can have a purple or white background depending on the background of the parent element. it would be confusing to write `.parent.exciting > .child.exciting { background : $plain}`